docs/migrations/version-2-to-3.md

```diff
@@ -2,6 +2,22 @@
 
 This document contains all the breaking changes and migrations guidelines for adapting your code to the new version.
 
+## allowInheritance set to true will enable inheritance
+
+This feature introduces a new option called `allowInheritance` in the interpreter options, which controls whether the generated models should inherit when the schema includes an `allOf`. By default, this option is set to false, which means that you'll not be affected if this property is not set. In the `MetaModel` and the `ConstrainedMetaModel` options, there is now an `extend` property (a list of models) and an `isExtended` property (boolean).
+
+Here is an example of how to use the new feature and the `allowInheritance` option in your code:
+
+```ts
+const generator = new JavaFileGenerator({
+  processorOptions: {
+    interpreter: {
+      allowInheritance: true
+    }
+  }
+});
+```
+
```

Comment:
Yea, if we show in the subsequent PRs how that code will look in the different generators, that would be nice 👌